sampleconfig/core.yaml

```diff
@@ -336,6 +336,27 @@ peer:
         #  - from:
         #    to:
 
+        # Config parameters for the Byzantine Fault Tolerant (BFT) client implementation
+        bft:
+            # Whether to use a BFT client implementation
+            enabled: false
+
+            # The minimal back-off delay. The minimal time between connection retries. This interval grows
+            # exponentially until maxBackoffDelay, below.
+            minBackoffDelay: 10ms
+
+            # The maximal back-off delay. The maximal time between connection retries.
+            maxBackoffDelay: 10s
```

Comment:
we have **reConnectBackoffThreshold** for this. Can we just use **reConnectBackoffThreshold** ?

Reply:
we could, but I think the default there is way off, so I opted to separate that.

Reply:
and there is a bug in reConnectBackoffThreshold  retrieval ;-)